discussions/secure_web_api.md

```diff
@@ -0,0 +1,119 @@
+# L3AFD Secure web API
+
+## Overview
+
+Walmart developed L3AF to simplify the management and orchestration of multiple eBPF programs in an enterprise
+environment.
+
+L3AF’s control plane consists of multiple components that work together to orchestrate eBPF programs:
+
+- L3AF Daemon (L3AFD), which runs on each node where eBPF program runs. L3AFD reads configuration data and manages 
+  the execution and monitoring of eBPF programs running on the node.
+- Deployment APIs, which a user calls to generate configuration data. This configuration data includes which eBPF 
```

Comment:
would help to clarify here what component exposes such APIs (is it L3AFD?), and whether these are local APIs or remote APIs.

```suggestion
- Deployment HTTP APIs exposed by L3AFD, which a user calls to generate configuration data. This configuration data includes which eBPF 
```